components/SplitDisplay.vue

```diff
@@ -9,7 +9,11 @@
           </h3>
           <p v-if="split.BestSegmentTime">
             <strong>Best time:</strong> {{ bestTimeDisplay }}
-            <b-button @click="fixGoldsModal" size="sm" variant="warning" class="ml-2">Fix fake golds</b-button>
+          </p>
+          <p>
+            <b-button @click="fixGoldsModal" size="sm" variant="warning">Fix fake golds</b-button>
+            <b-button v-if="isNotFirstSplit" @click="mergePreviousSplit" size="sm" variant="outline-primary" class="ml-2">Merge into previous split</b-button>
+            <b-button v-if="isNotLastSplit" @click="mergeNextSplit" size="sm" variant="outline-primary" class="ml-2">Merge into next split</b-button>
```

Comment:
Wouldn't "info" look better here? It's not a trick question choose the one you prefer, you can ignore suggestion if it doesn't look good
```suggestion
            <b-button v-if="isNotLastSplit" @click="mergeNextSplit" size="sm" variant="info" class="ml-2">Merge into next split</b-button>
```